migrations/1692188000000_brc20-deploys-ticker-index.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export function up(pgm: MigrationBuilder): void {
+  pgm.addColumns('brc20_deploys', {
+    ticker_lower: {
+      type: 'char(4)',
```

Comment:
👍🏻

Lol, there might even be disadvantages to `char`. I would have assumed that's optimized for (in addition to less storage), if we know the length, but postgres docs say otherwise 😅 ![image](https://github.com/hirosystems/ordinals-api/assets/6362150/c6f2f2b5-d82d-41be-97f7-584ea0c9664a)